workflows/variants.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: 'Variants'
+description: 'Learn how to create and use multiple variants for workflow steps in Novu.'
+icon: 'code-branch'
+---
+
+Variants is a workflow component that empowers users to create multiple versions of a given workflow step.
+
+You can use variants to ensure that only specific versions of your workflow step (e.g Email) are sent to subscribers. These variants are linked to specific conditions. No variant can exist without a condition. 
+
+Variants are an integral component of our [multi-tenancy feature](/tenants/introduction#tenant-specific-notification-content). 
+
+## Steps to create Variants
+
+In this case, we'll use a workflow Email step. Let's create multiple variants for this step.
+
+1. Create an email step in your workflow.
+
+<Frame caption="Email step in workflow"><img src="https://github.com/novuhq/docs/assets/2946769/b7d21890-10db-4452-ad95-2414976786be" /> </Frame>
```

Comment:
@unicodeveloper 
Have we uploaded these assets earlier?
Are we not storing these assets in images folder now?